test/test_helpers/integration-helpers.ts

```diff
@@ -156,33 +156,76 @@ export function normalizeToFs(files: RecursiveRecord<string | RecursiveRecord<st
   return newShape;
 }
 
+async function _buildResult(connection: MessageConnection, normalizedPath: string) {
+  const params = {
+    command: 'els.registerProjectPath',
+    arguments: [normalizedPath],
+  };
+
+  return (await connection.sendRequest(ExecuteCommandRequest.type, params)) as {
+    registry: Registry;
+  };
+}
+
 export async function createProject(
   files,
   connection: MessageConnection,
-  projectName?: string
-): Promise<{ normalizedPath: string; originalPath: string; result: UnknownResult; destroy(): void }> {
+  projectName?: string | string[],
+  config?: { local: { addons: string[]; ignoredProjects: string[] } }
+): Promise<{ normalizedPath: string | string[]; originalPath: string; result: UnknownResult | UnknownResult[]; destroy(): void }> {
   const dir = await createTempDir();
 
   dir.write(normalizeToFs(files));
-  const normalizedPath = projectName ? path.normalize(path.join(dir.path(), projectName)) : path.normalize(dir.path());
 
-  const params = {
-    command: 'els.registerProjectPath',
-    arguments: [normalizedPath],
-  };
+  if (config && Array.isArray(config.local.ignoredProjects)) {
+    const ignoredProjects = config.local.ignoredProjects;
 
-  const result = (await connection.sendRequest(ExecuteCommandRequest.type, params)) as {
-    registry: Registry;
-  };
+    for (let i = 0; i < ignoredProjects.length; i++) {
+      const ignoredProject = ignoredProjects[i];
+      const configParams = {
+        command: 'els.setConfig',
+        arguments: [{ local: { addons: [], ignoredProjects: [ignoredProject] } }],
+      };
```

Comment:
we should set config only once, because we have one server instance, 

```js
  const configParams = {
        command: 'els.setConfig',
        arguments: [{ local: { addons: [], ignoredProjects } }],
      };
```